operator/auth/cell.go

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package auth
+
+import (
+	"github.com/spf13/pflag"
+
+	"github.com/cilium/cilium/operator/auth/spire"
+	"github.com/cilium/cilium/pkg/hive/cell"
+)
+
+const (
+	MTlsEnabled = "mesh-auth-mtls-enabled"
```

Comment:
Can we name it `MTLSEnabled` the lowercase `ls` looks off to me... think we also did than in other places